ciao-launcher/qemu.go

```diff
@@ -358,7 +358,8 @@ func launchQemuWithNC(params []string, fds []*os.File, ipAddress string) (int, e
 		ncString := "socket,port=%d,host=%s,server,id=gnc0,server,nowait"
 		params[len(params)-1] = fmt.Sprintf(ncString, port, ipAddress)
 		var errStr string
-		errStr, err = qemu.LaunchQemu(context.Background(), params, fds, qmpGlogLogger{})
+
+		errStr, err = qemu.LaunchCustomQemu(context.Background(), "", params, fds, qmpGlogLogger{})
```

Comment:
I think this is okay.  We just need to document how it works in the qemu package.  The slice of fds passed to this array will be passed directly to Command.Extrafiles.  This means that the fds received inside qemu will be derived from the location of the parent fds within this slice.  As launcher has control over the array of fds and all the other parameters that it will pass to qemu, it can ensure that the two are in sync, i.e., that the fd 3 that it passes to qemu in a parameter is the first fd in the fd slice.  As long we document this we should be okay.